apps/assessment_registry/dashboard_schema.py

```diff
@@ -0,0 +1,188 @@
+from dataclasses import dataclass
+import itertools
+import graphene
+from .models import AssessmentRegistry, MethodologyAttribute
+from .enums import (
+    AssessmentRegistryCoordinationTypeEnum,
+    AssessmentRegistryDataCollectionTechniqueTypeEnum,
+)
+from django.db.models import Count
+from django.db import models
+from deep.caches import CacheHelper
+from collections import defaultdict
+
+
+def get_global_filters(_filter: dict, date_field="created_at"):
+    return {
+        f"{date_field}__gte": _filter["date_from"],
+        f"{date_field}__lte": _filter["date_to"],
+    }
+
+
+@dataclass
+class AssessmentDashboardStat:
+    cache_key: str
+    assessment_registry_qs: models.QuerySet
+    methodology_attribute_qs: models.QuerySet
+
+
+class AssessmentFilterInputType(graphene.InputObjectType):
```

Comment:
Let's use this term instead as this will include other filters as well in the future.
```suggestion
class AssessmentDashboardFilterInputType(graphene.InputObjectType):
```